sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/tasks/CreateNewTaskSteps.java

```diff
@@ -127,7 +144,7 @@ public void selectTaskStatus(String taskStatus) {
 
   public Task collectTaskData() {
     return Task.builder()
-        .taskContext("GENERAL")
+        .taskContext(getTaskContext())
```

Comment:
Please make sure that all the test cases which are using this step are still working after change